### DISCUSSION TITLE
D1

### DISCUSSION THREAD

**Discussion by rafaelg7** [9eu1/France-Project.github.io#6] [General]
@flxp229 
Hör doch jetzt auf ständig Verbesserungen vorzunehmen. Herr Schales hat unsere Verlinkung einfach vergessen. Weißt du, wenn du es lasst, können wir es auch archivieren.

**Comment by rafaelg7** [comment]
Kann ich es archivieren?
@flxp229

**Reply by ghost** [reply]
@rafaelg7 Sorry für die Verspätung. Ja, ist so‘n Problem mit mir. Wenn ich mal denke, ich hätte was gut hinbekommen, kann ich nicht aufhören, Fehler zu finden und sie auszubessern. Ich werd das Projekt jetzt archivieren.

**Reply by ghost** [reply]
Ich werd aber vorher noch kurz mit ein paar Bilder durch eigene oder lizenzfreier ersetzen, nur zur Sicherheit. Die Website wird ja sowieso keiner mehr besuchen.

**Reply by rafaelg7** [reply]
Okay, alles klar.
